packages/react-router/modules/matchPath.js

```diff
@@ -30,7 +30,8 @@ const matchPath = (pathname, options = {}) => {
   if (typeof options === 'string')
     options = { path: options }
 
-  const { path, exact = false, strict = false } = options
+  const { exact = false, strict = false } = options
+  const path = options.path || options.from
```

Comment:
> Then this opens up weird stuff like <Route from/> doesn't it?

Who cares? It's an implementation detail. It's private API, we don't document it, and if anyone does any thing like `<Route from>`, we don't support it.